src/platforms/java/common/performance/index.mdx

```diff
@@ -166,3 +166,59 @@ if (span != null) {
     }
 }
 ```
+
+### Connecting Errors with Spans
+
+Sentry errors can be linked with transactions and spans.
+
+Errors reported to Sentry while transaction or span is running are linked automatically:
+
+```java {tabTitle: Java}
+ISpan span = Sentry.startTransaction("task");
+try {
+    processItem();
+} catch (Exception e) {
+    Sentry.captureException(e);
+} finally {
+    span.finish();
+}
+```
+
+```kotlin {tabTitle: Kotlin}
+val span = Sentry.startTransaction("task")
+try {
+    processItem()
+} catch (e: Exception) {
+    Sentry.captureException(e)
+} finally {
+    span.finish()
+}
+```
+
+In cases where the decision if thrown exception should be reported to Sentry happens later in the future after span is already finished, to connect error with a span you must link the exception with a Span by calling `setThrowable` method:
+
+```java {tabTitle: Java}
+ISpan span = Sentry.getSpan();
+try {
+    processItem();
+} catch (Exception e) {
+    span.setThrowable(e);
+    throw e;
+} finally {
+    span.finish();
+}
+```
+
+```kotlin {tabTitle: Kotlin}
+val span = Sentry.getSpan()
+try {
+    processItem()
+} catch (e: Exception) {
+    span.setThrowable(e)
+    throw e
+} finally {
+    span.finish()
+}
+```
+
+Please note, that in Spring Boot integration, errors are linked with spans automatically and there is no need to follow these steps.
```

Comment:
Move this up earlier in the page so Spring Boot users don't get to the end, only to learn they didn't need to read this far :).
```suggestion
<Note>

In our Spring Boot integration, errors are linked with spans automatically.

</Note>
```